modules/reference/pages/systems/system-details/sd-software-channels.adoc

```diff
@@ -20,6 +19,22 @@ The child channels associated with the base channel are populated in the [guimen
 Mandatory child channels are pre-selected.
 Toggle the [guimienu]``include recommended`` switch to also select recommended child channels.
 
+
+
+[NOTE]
+====
+By default, cloned vendor channels match the original vendor channel and automatically select the dependencies.
+You can disable the automatic selection for cloned channels by adding the following option in [path]``/etc/rhn/rhn.conf``:
+----
+java.cloned_channel_auto_selection = false
+----
+====
+
+In general, channel pre-selection is necessary to avoid problems with missing dependencies for a given channel but it only works flawlessly for vendor channels.
+For cloned channels, {productname} relies on the channel label to define dependencies and use it in auto selection.
+This heuristic sometimes causes a side-effect of having unnecessary channels being automatically selected as dependencies.
+In such a case, it can be necessary to disable auto selection using the above mentioned setting.
```

Comment:
I will remove this line as we have already covered it in note.

Reply:
yes, makes sense!